game/requesthandler.hh

```diff
@@ -18,12 +19,37 @@ using Performous_Router_t = restinio::router::express_router_t<restinio::router:
 using Performous_Router_t = restinio::router::express_router_t<restinio::router::std_regex_engine_t>;
 #endif
 
-using Performous_Server_Traits = restinio::traits_t<
-         restinio::asio_timer_manager_t,
-         restinio::shared_performous_logger_t,
-         Performous_Router_t,
-         boost::asio::strand<boost::asio::executor>>;
-       
+class Performous_IP_Blocker {
+	public:
+		restinio::ip_blocker::inspection_result_t inspect(const restinio::ip_blocker::incoming_info_t& info) noexcept {
+			if (config["webserver/access"].i() != 3) return restinio::ip_blocker::allow(); // Just in case.
```

Comment:
Can this `3` be changed to an enumeration? i don't like magic numbers :)

Reply:
It is an enumeration but I believe all configuration enums are implemented as int underneath?

Reply:
Not sure, however an typed enum will just make it more readable, maybe not a part of this pr